schemas/context/enduserids.schema.json

```diff
@@ -9,42 +9,30 @@
   "$schema": "http://json-schema.org/draft-06/schema#",
   "title": "End User IDs",
   "type": "object",
-  "description": "Condensed, normalized encapsulation of all end user identifiers. NOTE: At least one of the fields is required.",
-  "minProperties": 1,
+  "description": "Condensed, normalized encapsulation of all end user identifiers. NOTE: At least one of the fields is required.\n",
   "properties": {
-    "xdm:realm": {
-      "title": "Realm",
-      "$ref": "https://ns.adobe.com/xdm/data/datasource",
-      "description": "The optional realm associated with the identity stitching strategy."
+    "https://ns.adobe.com/experience/mcId": {
+      "title": "Adobe Marketing Cloud Id",
```

Comment:
ID should be capitalized.

Reply:
Why?

Reply:
From https://github.com/adobe/xdm/blob/master/CONTRIBUTING.md#coding-styleguides
> Acronyms in camelCase like ID, API, JSON are also capitalized in camelCase, such as assetID

In this case: *ID* stands for Identifier, not for Sigmund Freud's concept of the "Id".

Reply:
Not to be pedantic, but Id is an abbreviation and not an acronym so it should not be cased as such. It should be `Id` not `ID`. This is what was agreed upon in the XDM style guide. 

When did we decide to deviate for what was agreed upon previously?

Reply:
I find the use of fully capitalized "ID" to be outside the industry norm. What was the reasoning for the change. Note this is not an Acronym, so is not covered by that change.

Reply:
Use of "ID" is called out specifically in contributing, and used on the other models (like asset).  I suggest for the purposes of this PR we follow the guidelines.  If we want to change them we should open an issue and have a separate discussion.

Reply:
@cfraser I love me a good pedantic remark: https://github.com/adobe/xdm/pull/131

As @kstreeter pointed out, we use `ID` elsewhere, so let's keep it consistent.

Reply:
@trieloff Ha! Touché, however, since we've agreed on pascal case `ID` doesn't conform, right?

I don't want to make this a semi-colon, no semi-colons debate but...